### PR TITLE
✨ Add test failing and flaking issue templates

### PR DESCRIPTION
**Pull request by mboukhalfa**
Adding templates to report CI failures

### CODE REVIEW

**Comment by mboukhalfa** [comment]
/assign @kashifest 
cc @furkatgofurov7

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/flaking-test.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Flaking Test
+description: Report flaky tests or jobs in Metal3 CI
+labels: kind/flake
```
[the remark this answers is not in the document]
Comment (mboukhalfa) [reply .github/ISSUE_TEMPLATE/flaking-test.yaml]: done

**Comment by mboukhalfa** [comment]
> /lgtm

@zhouhao3 Can you re lgtm!

**Review by furkatgofurov7 — APPROVED** [review]

**Comment by smoshiur1237** [comment]
\lgtm

**Pull request merged**